modules/pubmaticBidAdapter.js

```diff
@@ -275,6 +277,46 @@ function _createImpressionObject(bid, conf) {
   return impObj;
 }
 
+function _getDigiTrustObject(key) {
+  function getDigiTrustId() {
+    let digiTrustUser = window.DigiTrust && (config.getConfig('digiTrustId') || window.DigiTrust.getUser({member: key}));
+    return (digiTrustUser && digiTrustUser.success && digiTrustUser.identity) || null;
+  }
+  let digiTrustId = getDigiTrustId();
+  // Verify there is an ID and this user has not opted out
+  if (!digiTrustId || (digiTrustId.privacy && digiTrustId.privacy.optout)) {
+    return null;
+  }
+  return digiTrustId;
+}
+
+function _handleDigitrustId(eids) {
+  let digiTrustId = _getDigiTrustObject(PUBMATIC_DIGITRUST_KEY);
+  if (digiTrustId !== null) {
+    eids.push({
+      'source': 'digitru.st',
+      'uids': [
+        {
+          'id': digiTrustId.id || '',
+          'atype': 1,
+          'ext': {
+            'keyv': parseInt(digiTrustId.keyv) || 0
+          }
+        }
+      ]
+    });
+  }
+}
+
+function _handleEids(payload) {
+  let eids = [];
+  _handleDigitrustId(eids);
+  if (eids.length > 0) {
+    // todo: are we sure we have to send this data at user.eids or user.ext.each_eid_solution_object?
+    payload.user.eids = eids;
```

Comment:
To clarify - is this still an open question?  Or has it been confirmed?

If the latter, can you remove the comment?

Reply:
Hello @jsnellbaker , 
I have removed the comment now.